devito/logger.py

```diff
@@ -34,7 +35,8 @@
 logging.addLevelName(DSE, "DLE")
 logging.addLevelName(DSE_WARN, "DLE_WARN")
 
-logger.setLevel(INFO)
+init_log_level = eval(parameters["log_level"])
```

Comment:
You usually use the following to set the level of the log: `setLevel(INFO)`. `INFO` here is the *name* of the constant, not the value. If we are doing `setLevel(level)`, the variable `level` would need to contain the *value* of the constant `INFO` to achieve the same effect. This would require that a user of devito be expected to open the source of python's in-built `logging` module (since that is where we inherit the log-level constants from) to see what value the constant was set to and use that in our parameters file. The constant value can potentially change between python versions (which is the reason for using constants in the first place), and hence might produce unexpected results sometimes. Using `eval` we can use the constant names which are guaranteed to be consistent across versions and yet accept the log-level as a string (which is how it would come to us if the log-level was set in a config file or an environment variable).